tsconfig.paths.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "baseUrl": "./src",
+    "paths": {
```

Comment:
The dollar sign is just as valid as any other ASCII character to be sure, but to me it smacks of an angular practice.  See earlier comment about using the `@` for defining relative paths.